controller/workitem_blackbox_test.go

```diff
@@ -2481,3 +2583,40 @@ func (s *workItemChildSuite) TestWorkItemListFilterByNoParents() {
 	})
 
 }
+
+func createSpaceWithDefaults(ctx context.Context, db *gorm.DB) (*space.Space, *iteration.Iteration, *area.Area) {
+	spaceRepo := space.NewRepository(db)
+	iterationRepo := iteration.NewIterationRepository(db)
+	areaRepo := area.NewAreaRepository(db)
+
+	newSpace := space.Space{
+		Name: fmt.Sprintf("The Space %v", uuid.NewV4()),
+	}
+	sp, err := spaceRepo.Create(ctx, &newSpace)
+	if err != nil {
+		fmt.Println("Failed to create space for area.")
+		return nil, nil, nil
+	}
+
+	// root area
+	ar := &area.Area{
+		Name:    sp.Name,
+		SpaceID: sp.ID,
+	}
+	err = areaRepo.Create(ctx, ar)
+	if err != nil {
+		fmt.Println("Failed to create area.")
```

Comment:
that isn't right! we have a package log inside almighty-core. We use this everywhere, have a look at log.Error(ctx, map[string]interface{"area_name": ar.Name, "err": err}, "failed to create area.")`